### PR TITLE
Knowledge base discoverability improvements

### LINKED ISSUE #18934: Matomo knowledgebase discoverability improvements
In order to assist with our goal of improving discoverability of relevant, timely information while onboarding and using Matomo we have the following improvements:

## Summary

- [ ] adding a Search placeholder so it’s more clear there is a search.

image

- [ ] In the search box make the categories bold to highlight them better and people better noticing the “help resources”

image (1)

- [ ] On the info page shows the h4 headers bigger to better see the different versions of help like Community help. And we could also mention a benefit around it like find what you are searching for quickly or discover new features by 
searching or what not. Would need to come up with something

image (2)

- [ ] Add links to the relevant guides where possible, eg there is currently no link to the Goals guide.

image (4)

### PR DESCRIPTION
**Pull request by justinvelluppillai**
### Description:

Resolve #18934 - adds UI changes to make KB search more obvious; different KB help types more obvious; and adds links to relevant User guides for Goals and Ecommerce tracking.

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

File: plugins/CoreHome/vue/src/QuickAccess/QuickAccess.vue [hunk 1]
```diff
@@ -22,6 +21,7 @@
       v-model="searchTerm"
       type="text"
       tabindex="2"
+      placeholder="Search"
```
Comment (sgiehl) [inline plugins/CoreHome/vue/src/QuickAccess/QuickAccess.vue]: Would be good to use a translation here instead.

**Comment by justinvelluppillai** [comment]
Ready for review. I have left the UI tests failing to make it easier to see the differences. I will update them on approval.

**Review by sgiehl — COMMENTED** [review]
The code changes are looking fine. But shouldn't we maybe increase the contrast of the new placeholder `Search` text a bit? That one is really hard to read.

**Comment by justinvelluppillai** [comment]
I guess the `#d1d1d1` comes from materialize or another include and is used elsewhere. I'll override it for this to `#a9a9a9`

**Review by peterhashair — APPROVED** [review]

**Comment by peterhashair** [comment]
Coupe screenshots maybe need an update, but other than that looks good to me.

**Pull request merged**